redisgraph/query_result.py

```diff
@@ -8,8 +8,8 @@
 class ResultSetColumnTypes(object):
     COLUMN_UNKNOWN = 0
     COLUMN_SCALAR = 1
-    COLUMN_NODE = 2
-    COLUMN_RELATION = 3
+    COLUMN_NODE = 2       # Unused as of RedisGraph v2.1.0, retained for backwards compatibility.
+    COLUMN_RELATION = 3   # Unused as of RedisGraph v2.1.0, retained for backwards compatibility.
```

Comment:
I've been using 2.1.0 as a placeholder because I think it is a sensible choice, but can rewrite these as 2.0.11 if preferred.